tests/torch/pytorch_patch_isolated.py

```diff
@@ -80,14 +80,24 @@ def test_jit_script_exception_preserves_patching_isolated():
 
 
 def compile_and_run_test_model() -> torch.Tensor:
-    from tests.torch.helpers import BasicConvTestModel
```

Comment:
Had to drop `BasicConvTestModel` because during its import `nncf.torch` is imported which is not acceptable for this specific test